routes/htmlRoutes.js

```diff
@@ -1,14 +1,12 @@
 const path = require("path");
 const express = require("express");
 
-const db = require("../models");
-
 module.exports = function(app) {
-  // Load index page
-  app.use("/", express.static(path.resolve("public")));
```

Comment:
This make the index.html file available as well as everything else under the public folder (eg: images)